cardano_node_tests/utils/dbsync_utils.py

```diff
@@ -784,10 +784,9 @@ def check_tx(
         f"({response.invalid_hereafter} != {tx_raw_output.invalid_hereafter})"
     )
 
-    combined_txins: Set[clusterlib.UTXOData] = {
-        *tx_raw_output.txins,
-        *[p.txins[0] for p in tx_raw_output.script_txins if p.txins],
-    }
+    combined_txins = tx_raw_output.txins + [
+        p.txins[0] for p in tx_raw_output.script_txins if p.txins
+    ]
```

Comment:
Optional: I like the original variant with unpacking better (the original code can be just changed from set to list) because
a. it is obvious what is the resulting data type
b. works for any iterable - I don't need to make sure all components are of the same type, it can concat e.g. set with tuple and list